runtime/doc/terminal.txt

```diff
@@ -187,26 +187,40 @@ event. The event is handled directly by Nvim and is not forwarded to plugins.
 
 OSC 133: shell integration			*terminal-osc133*
 
-Some shells will emit semantic escape sequences (OSC 133) to mark the
-beginning and end of a prompt. The start of a prompt is marked with the
-sequence `OSC 133 ; A`. Nvim can be configured to create signs in terminal
-buffers marking shell prompts. Example: >lua
+Shells can emit semantic escape sequences (OSC 133) to mark where each prompt
+starts and ends. The start of a prompt is marked by sequence `OSC 133 ; A ST`,
+and the end by `OSC 133 ; B ST`.
+
+                                                *shell-prompt-config*
+To configure bash to mark the start/end of each prompt, set $PROMPT_COMMAND
+and $PS1 as follows: >bash
+
+    # Prompt start:
+    PROMPT_COMMAND='printf "\033]133;A\007"'
+    # Prompt end:
+    PS1=$PS1'\033]133;B\007'
```

Comment:
We should have clear docs about how to enable this in shells. Starting with bash, but can add more.

Reply:
Fish 4.0 includes this by default now, so nothing needed on our end :)

A lot of terminals (iTerm2, kitty, Ghostty, perhaps others) also bundle their own shell integration scripts for different shells. In those cases, the terminal sets it up and the user doesn't need to do anything differently. Might be worth mentioning this as well.

Reply:
I'm using kitty but I guess I didn't run enable shell integration thing it has.